api/api.py

```diff
@@ -197,18 +196,12 @@ def dispatcher(router, request, response):
             response.headers['Content-Type'] = 'application/json; charset=utf-8'
     except webapp2.HTTPException as e:
         util.send_json_http_exception(response, str(e), e.code)
-    except Exception as e: # pylint: disable=broad-except
-        if config.get_item('core', 'debug'):
-            message = traceback.format_exc()
-        else:
-            message = 'Internal Server Error'
-        util.send_json_http_exception(response, message, 500)
```

Comment:
As long as there is there is defined behaviour for `Exception` bubbling up I'm fine with it :+1: